Backend/microservices/data/src/main/java/com/ms/data/service/XMLService.java

```diff
@@ -0,0 +1,30 @@
+package com.ms.data.service;
+
+import com.ms.data.model.XmlObject;
+import org.springframework.stereotype.Service;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+import java.io.StringReader;
+
+
+@Service
+
+public class XmlService {
```

Comment:
![image](https://user-images.githubusercontent.com/75213424/160245787-921118f7-3d73-44e0-9bf4-121ffd52d497.png)
change name of file to XmlService.java